regolith/helpers/a_manurevhelper.py

```diff
@@ -0,0 +1,125 @@
+"""Builder for Current and Pending Reports."""
+import datetime as dt
+import sys
+import time
+from argparse import RawTextHelpFormatter
+
+import nameparser
+
+from regolith.helpers.basehelper import SoutHelperBase, DbHelperBase
+from regolith.dates import month_to_int, month_to_str_int
+from regolith.fsclient import _id_key
+from regolith.sorters import position_key
+from regolith.tools import (
+    all_docs_from_collection,
+    filter_grants,
+    fuzzy_retrieval,
+)
+
+ALLOWED_TYPES = ["nsf", "doe", "other"]
+ALLOWED_STATI = ["invited", "accepted", "declined", "downloaded", "inprogress",
+                 "submitted", "cancelled"]
+
+
+def subparser(subpi):
+    subpi.add_argument("name", help="Full name of the first author",
+                       )
+    subpi.add_argument("due_date", help="due date in form YYYY-MM-DD in quotes", default=''
+                       )
+    subpi.add_argument("-c", "--recommendation",
+                       help="Recommendation for this manuscript: reject, smalledits, or majoredits", default=None
+                       )
+    subpi.add_argument("-d", "--database",
+                       help="The database that will be updated. Defaults to "
+                            "first database in the regolithrc.json file."
+                       )
+    subpi.add_argument("-r", "--reviewer",
```

Comment:
I wrote a comment but lost it somehow.  Let's remove me as being hard-coded here and take it from the user regolith config if it exists there and throw an exception.   Jiawei (I think) knows how to do this.

Reply:
The new commit has been updated with your suggestions except for this one. I need to contact Jiawei to figure it out.